### PR TITLE
Fuse Repayment DAO Vote

### PR DESCRIPTION
**Pull request by xklob**
# PR Type: Feature

# PR Title: Fuse Repayment DAO Vote

### PR Checklist - Feature (Proposal)

- [x] All Tests Passing
- [x] Proposal Added to ProposalsConfig
- [x] Fork Block Correct
- [x] Remove Any .only's on Tests
- [x] Update Documentation If Needed
- [x] Update Roles Config
- [x] Update Proposal Description
- [x] Generate Calldata
- [x] Proposal Submitted

### CODE REVIEW

**Review by thomas-waite — DISMISSED** [review]
Looks good. Approved pending tests passing

**Comment by xklob** [comment]
Disabled a bunch of broken/fragile e2es. Some remaining that I'm unsure how to fix or if I should disable them.

File: proposals/description/tip_112.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+First, the strategy for repayment will be staged by asset. The affected assets are:
+
+token │ amount stolen │ amount stolen usd
+
+─────────┼─────────────────┼─────────────────────
+
+FEI │ 20,251,603.12 │ $20,361,973.83
+
+ETH │ 6,037.91 │ $17,062,057.88
```
Comment (xklob) [inline proposals/description/tip_112.ts]: * change to $7m / recalculate

File: proposals/description/tip_112.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+TIP-112: Fuse Repayment
+
+Summary: Whitelists the “Fuse Repayment Contract” as a safe PCV address for the Tribal Council or PCV Guardian to repay.
+
+Proposal:
+
+The community has signaled support for a PCV repayment of the Fuse exploit in a previous snapshot. 
+This proposal provides some technical details and associated timelines.
+
+First, the strategy for repayment will be staged by asset. The affected assets are:
```
Comment (xklob) [inline proposals/description/tip_112.ts]: remove this through line 44

**Pull request merged**